smurfs/src/components/MainForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import axios from 'axios'
+// import { useContext } from 'react'
+// import { SmurfContext } from '../contexts/SmurfContext.js'
+
+export default class MainForm extends React.Component {
```

Comment:
i suggest using functional components across the board if you are going to use hooks.